src/plugins/WebOfTrust/WebOfTrust.java

```diff
@@ -448,6 +450,15 @@ File getUserDataDirectory() {
 
 	private com.db4o.config.Configuration getNewDatabaseConfiguration(boolean readOnly) {
 		com.db4o.config.Configuration cfg = Db4o.newConfiguration();
+		// use cached io
+		/** Caching store access 
+		 * 
+		 * When you call commit() then db4o does a flush anyway to ensure that the commit is persistent. 
+		 * http://community.versant.com/Forums/tabid/98/aft/10198/Default.aspx
```

Comment:
This link is dead nowadays, and also not available on archive.org / archive.is :(